mkdocs.yml

```diff
@@ -6,7 +6,7 @@ site_author: DAOS Project
 # Repository
 repo_name: daos-stack/daos
 repo_url: https://github.com/daos-stack/daos
-#edit_uri: blob/release/2.4/docs/
+# edit_uri: blob/release/2.4/docs/
```

Comment:
I had to include this fix as my pre-commit checks failed on it otherwise.

Reply:
gosh ... i guess i have to be more careful with comments in yml files

Reply:
No problem.  There's a commit-hook to add pre-commit checking but that's blocked on https://github.com/daos-stack/daos/pull/8155 and your PR seemed to go through when github-actions were off-line so it didn't get checked, and it was a Doc-only change so it was force-landed so we didn't notice the missing checks.